docs/site/Getting-started.md

```diff
@@ -63,9 +63,22 @@ Now that we have a basic project created, it's time to add our own
 [controller](Controllers.md). Let's add a simple "Hello World" controller as
 follows:
 
-- Create a new file in `/src/controllers/` called `hello.controller.ts`.
+```sh
+lb4 controller
+```
+
+Answer the prompts as follows:
+
+```sh
+? Controller class name: Hello
+? What kind of controller would you like to generate? Empty Controller
```

Comment:
Did you explore the possibility of creating a REST controller? It would give us a lot more generated code the users don't have to copy/paste.

Reply:
Umm ... if you follow the tutorial, we create a model, a datasource and a repository before getting to creating a controller - if you're seeing the error despite the tutorial instructions then we have a bug with model detection in the CLI and it needs to be investigated as such.

Reply:
If i understand correctly, this page is the first app that you'd build to get started, not the to-do tutorial.  http://loopback.io/doc/en/lb4/Getting-started.html

Reply:
Oh, my bad ... for some reason I thought this was a part of example-todo ... which needs to be updated as well to use the lb4 controller command.

Reply:
let me create a separate task / PR for that. thx for noticing it!